tests/api/test_topic_entity_tag.py

```diff
@@ -128,8 +128,12 @@ def test_patch(self, test_topic_entity_tag, auth_headers): # noqa
                 "topic": "new_topic",
                 "entity_type": "new_type",
                 "entity": "new_entity",
-                "updated_by": "new_user",
-                "novel_topic_data": False,
+                "entity_id_validation": "alliance",
```

Comment:
There's no additional testing for these fields in the response.  Does patch now require adding all fields ?  Before we'd use PUT for that, and we used PATCH because it didn't require everything

Reply:
Good catch! We don't need that, but I leave as is for now since it will not affect the test (too lazy to change) =)

Reply:
Cool cool :)